### ISSUE TITLE
Animations with no duration set aren't executed as per spec

### ISSUE THREAD

**Issue by gribelu** [web-animations/web-animations-js-legacy#568]
Hello,

The spec says that when an animation has no duration set, the properties are set on the target element with no interpolation: http://dev.w3.org/fxtf/web-animations/#creating-a-new-animation-object
The polyfill only does this when explicitly setting duration to 0.

Also, should this be the same when using an EffectCallback? The spec isn't clear on this. When setting duration to 0 the callback isn't executed at all.

Thanks!

**Comment by alancutter** [comment]
I wonder if this is related to the default fill mode for animation being none. Do you have an example that shows the issue and describe what you expected to happen?

**Comment by gribelu** [comment]
Indeed, explicitly setting the fill mode to 'forwards' will apply the properties. But why is the default fill 'none'? The part of the spec I was referring to says:

> It is also possible to omit the timing parameter altogether in which case default timing values will be used. Since the intrinsic iteration duration of an animation is zero, and the default fill when constructing an Animation is forwards, it is possible to create animations that simply set a property without any interpolation as follows,
> 
> ``` javascript
> new Animation(elem, { display: 'none' });
> ```

I also tested this using an EffectCallback and the effect is the same. If setting fill mode to forwards the callback is executed just once.

I played with it in here if you want to see an example: http://jsfiddle.net/cwunF/

**Comment by ewilligers** [comment]
> the default fill when constructing an Animation is forwards

That part of the spec has been changed. Now the default for an animation is no fill.
